salt/metalk8s/orchestrate/backup/cleanup.sls

```diff
@@ -0,0 +1,19 @@
+{%- from "metalk8s/repo/macro.sls" import build_image_name with context %}
+
+{%- set master_nodes = salt.metalk8s.minions_by_role('master') %}
+{%- set image = build_image_name("metalk8s-utils") %}
+
+{%- set backup_folder = "/toto" %}
+
+{%- for node in master_nodes | sort %}
+
+Schedule backup cleanup Job on {{ node }}:
+  metalk8s_kubernetes.object_present:
+    - name: salt://{{ slspath }}/files/cleanup_job.yaml.j2
+    - template: jinja
+    - defaults:
+        files: __slot__:salt:metalk8s_backups.get_backups_to_delete("{{ backup_folder }}", 5)
+        node: {{ node }}
+        image: {{ image }}
```

Comment:
No, do not use a job just to cleanup the archive please, a simple salt state to remove those files everywhere will be good

(NOTE: We use a Job to copy the backup since it's a copy that may take a bit of time and we do not want to use salt to do such copy)

IMHO: There are 2 options, either
- You update the actual Job that copies the backup so that it removes old ones at the end
- You update the `salt/metalk8s/orchestrate/backup/replication.sls` orchestrate to remove old backup using salt (in this case the removal will be done before the copy but to me it's fine)